corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -365,7 +365,10 @@ hqDefine("users/js/mobile_workers",[
             }
 
             if (self.phoneStatusMessage()) {
+                $("#phone-help").hide();
```

Comment:
Can't you do this by putting a `data-bind=visible: phoneStatusMessage` directly on the div? Mixing jquery and knockout control gets confusing.

Reply:
Thanks for the tip. I'll check this out.